system/modules/isotope/library/Isotope/Model/Gallery/Standard.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Isotope\Model\Gallery;
 
+use Isotope\CompatibilityHelper;
```

Comment:
Namespaces are usually sorted by name. My IDE (PHPStorm) does that automatically, did you manually add these?

Reply:
not sure there is such an option. If you manually added the imports, try adding them in alphabetical order. In PHPStorm there's an option to "Optimize Imports" which removes unused (as in your previous PR) and also orders them automatically.